### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by vzdrizhni**
- Added handleClick function to complete the project.

### CODE REVIEW

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @vzdrizhni , 

calc

I stopped at MVP testing phase, because at `milestone 5`, the app should actually work
so I tried to solve `3 + 2`, it rendered `34`

Please make this work (addition, substration, multiplication, division, etc..)

Let's get to debugging....

Happy coding

**Comment by vzdrizhni** [comment]
Fixed the bug.

File: src/logic/operate.js [hunk 1]
```diff
@@ -7,29 +7,34 @@ const operate = (numberOne, numberTwo, operation) => {
 
   switch (operation) {
     case '÷': {
-      total = num1.div(num2);
-      return total;
+      if (num2 == 0) { // eslint-disable-line eqeqeq
+        total = 0;
```
Comment (shloch) [inline src/logic/operate.js:11]: `total` should be equal to `Error` or `NaN`...something like that
(`if num2 == 0`)

Also while disable this `===` operator ?

**Review by shloch — APPROVED** [review]
## Hello  @vzdrizhni 

Great job, this works smoothly this time............keep up with the good work
calc

### [optional changes]
- You can center the calculator with `margin:auto`
Screen Shot 2020-08-20 at 7 55 04 PM

YOU GET A THUMBS UP.......but Checkout the comment below

Happy coding

File: src/logic/operate.js (hunk 1 above)
Comment (vzdrizhni) [reply src/logic/operate.js:11]: Thanks for your review. To be honest, I already don't remember why I did it, but I remember that there should be a reason. 😄

**Pull request merged**